core/templates/dev/head/pages/library/Library.js

```diff
@@ -156,6 +156,7 @@ oppia.controller('Library', [
         // elements flush left.
         // Transforms the group names into translation ids
         $scope.leftmostCardIndices = [];
+        var i;
```

Comment:
Put var in the for loop instead? Ditto below. You could use different vars too (i for one, j for another).

Reply:
Done.